### PR TITLE
add maintainers rules and list

### PR DESCRIPTION
**Pull request by gnawux**
The rules are copied from containerd project with some minor modification:

- change names
- removed subproject rules

Signed-off-by: Wang Xu <gnawux@gmail.com>

### CODE REVIEW

**Comment by gnawux** [comment]
To introduce more community contributors and maintainers, we need specify these rules. Hence we borrow the rules from the containerd project. However, I am not sure should we have a "*Benevolent Dictator for Life*". 

Please comment here, guys.

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+	[people.gnawux]
+	Name = "Xu Wang"
+	Email = "gnawux@gmail.com"
+	GitHub = "gnawux"
+
+	[People.laijs]
+	Name = "Jiangshan Lai"
+	Email = "jiangshanlai@gmail.com"
+	GitHub = "estesp"
```
Comment (Crazykev) [inline MAINTAINERS]: Should be typo here, "estesp"->"laijs"

**Comment by gnawux** [comment]
update the rules, eliminate the "*Benevolent Dictator for Life*". I don't like to step onto this role.

Instead, we employ simple reviewers and approvers.

Now I need one reviewer and one approver to merge it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+solutions.
+
+Typically, the best methods of accomplishing this are to submit an issue,
+stating the problem. This issue can include a problem statement and a
+checklist with requirements. If solutions are proposed, alternatives should be
+listed and eliminated. Even if the criteria for elimination of a solution is
+frivolous, say so.
+
+Larger changes typically work best with design documents, similar to those found
+in `design/`. These are focused on providing context to the design at the time
```
Comment (bergwolf) [inline CONTRIBUTING.md]: well, we don't have a `design/` directory...

Comment (gnawux) [inline CONTRIBUTING.md]: I think I added one in this commit....

Comment (bergwolf) [inline CONTRIBUTING.md]: oh indeed!

**Pull request merged**